client/locations-list-map.js

```diff
@@ -1579,53 +1576,54 @@ function initMap(data, filters) {
 
   loadOtherCountries();
 
-  // Add map control for custom fullscreen behavior.
-  // (Regular fullsceen behavior of Google maps messes up Bootstrap modals, popovers, etc.)
-  gMap.controls[google.maps.ControlPosition.TOP_RIGHT].push(document.getElementById('customFullscreenButton'));
-  $('#customFullscreenButton').show();
-
-  $('#customFullscreenButton').on('click', () => {
-    // From https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_fullscreen2
-    if (document.fullscreenElement) {
-      if (document.exitFullscreen) {
-        document.exitFullscreen();
-      } else if (document.mozCancelFullScreen) { /* Firefox */
-        document.mozCancelFullScreen();
-      } else if (document.webkitExitFullscreen) { /* Chrome, Safari and Opera */
-        document.webkitExitFullscreen();
-      } else if (document.msExitFullscreen) { /* IE/Edge */
-        document.msExitFullscreen();
-      }
-    } else {
-      const elem = document.documentElement;
-      if (elem.requestFullscreen) {
-        elem.requestFullscreen();
-      } else if (elem.mozRequestFullScreen) { /* Firefox */
-        elem.mozRequestFullScreen();
-      } else if (elem.webkitRequestFullscreen) { /* Chrome, Safari and Opera */
-        elem.webkitRequestFullscreen();
-      } else if (elem.msRequestFullscreen) { /* IE/Edge */
-        elem.msRequestFullscreen();
+  if (!isEmbed) {
```

Comment:
@nburt The embed page does not show the map full screen control. Has this always been true, or did I remove it when I implemented the custom control? (Early on, I was just working with the home page and not really aware of the embed.)

Reply:
hmm I hadn't noticed that, I'm guessing it was left out when you did the custom embed. Any idea why it isn't working there?

Reply:
it's quite plausible I lost it when I did the UX2 redesign of the embed... whoops! sorry!